examples/kubernetes-namespaces/README.md

```diff
@@ -96,8 +97,8 @@ To demonstrate this, let's spin up a simple replication controller and pod in th
 The first step is to define a context for the kubectl client to work in each namespace.
 
 ```shell
-$ cluster/kubectl.sh config set-context dev --namespace=development
-$ cluster/kubectl.sh config set-context prod --namespace=production
+$ cluster/kubectl.sh config set-context dev --namespace=development --cluster=YourClusterName --user=YourUserName
```

Comment:
I wish there was a copy and edit context flow.

I think you can use your cluster name here as an example instead of using YourClusterName
We do the same thing, for example, in the output for kubetctl get rc.
The host there is your host name. It will be different for different users.
For clarity, you can show the output of kubectl config view first and then say that the values of --cluster and --user are picked from there.